MAUI/PDF-Viewer/Save-a-Document.md

```diff
@@ -27,4 +27,38 @@ private void SaveDocument()
 {% endhighlight %}
 {% endtabs %}
 
-To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
\ No newline at end of file
+To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
+
+## Flatten annotations and form fields on save
+
+Flattening refers to the process of converting interactive elements, such as annotations and form fields, into a static, non-editable format within a PDF document. The SfPdfViewer allows you to save the annotations and form fields by Flatten using `FlattenOnSave` API.
+
+### Flatten annotation on save
+
+The FlattenOnSave API helps you to flatten the specified annotation in a PDF document. Refer to the following code example:
```

Comment:
<img width="402" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/e5b2e219-8cea-47ae-9d6f-605556864170">

Mention what the following example states. It looks first annotation is flattened. You can find such references in other pages